src/widgetastic_patternfly4/drawer.py

```diff
@@ -0,0 +1,33 @@
+from widgetastic.utils import ParametrizedLocator
+from widgetastic.widget import Widget
+
+
+class BaseDrawer:
+    """Represents drawer component.
+
+    https://www.patternfly.org/v4/components/drawer
+    """
+
+    CLOSE = ".//button[@aria-label='Close drawer panel']"
+
+    @property
+    def is_open(self):
+        """Returns True if the Drawer panel is open"""
+        return "pf-m-expanded" in self.browser.classes(self)
+
+    def close(self):
+        """Close drawer."""
+        if self.is_open:
+            self.browser.click(self.browser.element(self.CLOSE))
+
+
+class Drawer(BaseDrawer, Widget):
```

Comment:
The drawer is not a widget but View. 
It can hold child widgets so let's move it to View.

Users can use it like

from widgetastic_platterfly4 import Drower

```
class FooDrower(Drower):
    ROOT = "If they want to overwrite it."
    foo = Foo(....)
    bar = Bar(....)
```

What do you think @LightOfHeaven1994 ?

We had a `Model` widget which maybe similar to it. Please check this PR
https://github.com/RedHatQE/widgetastic.patternfly4/pull/78

Reply:
@digitronik I thought about it too, but drawer is a bit different that a Modal. Modal looks like a separate view as it appears only when you open it. But Drawer is always there according to the DOM structure. 
Structure is:
```
<div class="pf-c-drawer">
  <div content> // this one is always visible
  <div panel> // this one opens only when we click on toggle
```
So maybe it makes sense to leave it as a widget. WDYT?

![image](https://user-images.githubusercontent.com/33912805/234237694-79f45185-9791-47c5-a87d-5c275d8a72d3.png)

Reply:
If the dower holds only text then we can add a read method and continue with a widget. 
but if it can hold other widgets then definitely we need to use View.

Reply:
@digitronik fixed and covered with tests both cases (basic drawer and custom one). Thanks for the advice :)